plugins/basemap-openstreetmap.js

```diff
@@ -4,32 +4,53 @@
 // @version        0.1.2
 // @description    Add the native OpenStreetMap.org map tiles as an optional layer.
 
-/* exported setup --eslint */
+/* exported setup, changelog --eslint */
 /* global L, layerChooser */
+
 // use own namespace for plugin
 var mapOpenStreetMap = {};
+window.plugin.mapOpenStreetMap = mapOpenStreetMap;
 
-mapOpenStreetMap.addLayer = function () {
-  // OpenStreetMap tiles - we shouldn't use these by default - https://wiki.openstreetmap.org/wiki/Tile_usage_policy
-  // "Heavy use (e.g. distributing an app that uses tiles from openstreetmap.org) is forbidden without prior permission from the System Administrators"
+var changelog = [
+  {
+    version: '0.1.3',
+    changes: ['Update OSM tile provider', 'Add CyclOSM tiles', 'Expose config'],
+  },
+];
 
-  var osmOpt = {
-    attribution: 'Map data © OpenStreetMap contributors',
-    maxNativeZoom: 18,
-    maxZoom: 21,
-  };
+// https://wiki.openstreetmap.org/wiki/Raster_tile_providers
 
-  var layers = {
-    'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png': 'OpenStreetMap',
-    'https://{s}.tile.openstreetmap.fr/hot/{z}/{x}/{y}.png': 'Humanitarian',
-  };
-
-  for (var url in layers) {
-    var layer = new L.TileLayer(url, osmOpt);
-    layerChooser.addBaseLayer(layer, layers[url]);
-  }
+// Common options
+var osmOpt = {
+  attribution: 'Map data © OpenStreetMap contributors',
+  maxNativeZoom: 18,
+  maxZoom: 21,
 };
 
+mapOpenStreetMap.LAYERS = [
+  {
+    name: 'OpenStreetMap',
+    url: 'https://tile.openstreetmap.org/{z}/{x}/{y}.png',
+    options: osmOpt,
+  },
+  {
+    name: 'Humanitarian',
+    url: 'https://{s}.tile.openstreetmap.fr/hot/{z}/{x}/{y}.png',
+    options: osmOpt,
+  },
+  {
+    name: 'CyclOSM',
+    url: 'https://{s}.tile-cyclosm.openstreetmap.fr/cyclosm/{z}/{x}/{y}.png',
+    options: osmOpt,
```

Comment:
```suggestion
    options: Object.assign({}, osmOpt),
```

... otherwise it's not possible to override only one of the layer settings as they share the same "options" instance

Reply:
it's possible by overriding the option object the same way

Reply:
That isn't very straightforward or user friendly, is it?

Reply:
I assume this is dev friendly enough
also, `Object.assign` is not ES5

Reply:
Es6 is supported by all browsers since 2017 (some even earlier).

Also, Object.assign is already used in our code (utils_file.js#135)

Reply:
grmpf, then coding style need to be updated